cmd/user-agent/src/pages/chapi/wallet/didcomm/blindedRouter.js

```diff
@@ -46,7 +45,13 @@ export class BlindedRouter {
 
         // send message to router requesting peerDID
         // TODO response peer DID from previous step to be sent to router
-        this.mediator.requestDID()
+
+        // TODO temporarily send myDID to test response (to test e2e)
```

Comment:
Issue number?

Reply:
no issue here, I added it to test locally, in next task #409 , this method will be gone

Reply:
looks like this comment should be on line 52

Reply:
rightnow, this function is not getting called since blinded routing is disabled in ENV